### ISSUE TITLE
/bin/sh -c go get github.com/golang/lint/golint  returned a non-zoro code :1

### ISSUE THREAD

**Issue by yankunsam** [rancher/os#1871]
**RancherOS Version: (ros os version)**

**Where are you running RancherOS? (docker-machine, AWS, GCE, baremetal, etc.)**
When I try run "make" , it shows Step 49/54.

**Comment by yankunsam** [comment]
I am sorry. It is because that I have not installed golang.

**Comment by yankunsam** [comment]
Step 49/54 : RUN go get github.com/golang/lint/golint
 ---> Running in 296705f8f209
package golang.org/x/tools/go/gcexportdata: unrecognized import path "golang.org/x/tools/go/gcexportdata" (https fetch: Get https://golang.org/x/tools/go/gcexportdata?go-get=1: dial tcp 216.239.37.1:443: i/o timeout)
The command '/bin/sh -c go get github.com/golang/lint/golint' returned a non-zero code: 1
FATA[0019] exit status 1
make: *** [default] Error 1


It appear again and I have installed golang.

**Comment by SvenDowideit** [comment]
Sounds like you're having network issues.

**Comment by yankunsam** [comment]
Yes.Because I am in China. Maybe I should download golint individually.

**Comment by yankunsam** [comment]
I have checked "/home/sam/go/src/golang.org/x/tools/go/gcexportdata" and it exist. But when run "make" .The error is the same. So what should I do ?

"package golang.org/x/tools/go/gcexportdata: unrecognized import path "golang.org/x/tools/go/gcexportdata" (https fetch: Get https://golang.org/x/tools/go/gcexportdata?go-get=1: dial tcp 216.239.37.1:443: i/o timeout"

**Comment by strowi** [comment]
Hi,

ran into the same problem, seems the culprit might be this: https://github.com/golang/lint/commit/9a272034dedb2a3ed05231d5604ce17fb40f0e58

Try `go get golang.org/x/lint/golint` instead .
